ArticleTemplates/articleTemplateContainerFeatures.html

```diff
@@ -44,7 +44,7 @@ <h1 class="headline selectable" id="headline">__TITLE__ __REVIEW_RATING__</h1>
         __SPONSORSHIP__
 
         <div class="article__body" id="feature-body">
-            <div class="from-content-api prose resizable selectable" id="feature-body-blocks">
+            <div class="from-content-api prose resizable resizable_line_height selectable" id="feature-body-blocks">
```

Comment:
Can you rename any references to the class `resizable_line_height` to `resizable-line-height`? (Single hyphens should be used as word separators in class names).